providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/FlagdProviderTest.java

```diff
@@ -470,143 +459,145 @@ void reason_mapped_correctly_if_unknown() {
-        OpenFeatureAPI.getInstance().setProviderAndWait(provider);
-
-        HashMap<String, com.google.protobuf.Value> flagsMap = new HashMap<String, com.google.protobuf.Value>();
-        HashMap<String, com.google.protobuf.Value> structMap = new HashMap<String, com.google.protobuf.Value>();
-
-        flagsMap.put(FLAG_KEY_BOOLEAN, com.google.protobuf.Value.newBuilder().setStringValue("foo").build());
-        flagsMap.put(FLAG_KEY_STRING, com.google.protobuf.Value.newBuilder().setStringValue("foo").build());
-        flagsMap.put(FLAG_KEY_INTEGER, com.google.protobuf.Value.newBuilder().setStringValue("foo").build());
-        flagsMap.put(FLAG_KEY_DOUBLE, com.google.protobuf.Value.newBuilder().setStringValue("foo").build());
-        flagsMap.put(FLAG_KEY_OBJECT, com.google.protobuf.Value.newBuilder().setStringValue("foo").build());
-
-        structMap.put("flags", com.google.protobuf.Value.newBuilder()
-                .setStructValue(Struct.newBuilder().putAllFields(flagsMap)).build());
-
-        // should cache results
-        FlagEvaluationDetails<Boolean> booleanDetails;
-        FlagEvaluationDetails<String> stringDetails;
-        FlagEvaluationDetails<Integer> intDetails;
-        FlagEvaluationDetails<Double> floatDetails;
-        FlagEvaluationDetails<Value> objectDetails;
-
-        // assert cache has been invalidated
-        booleanDetails = api.getClient().getBooleanDetails(FLAG_KEY_BOOLEAN, false);
-        assertTrue(booleanDetails.getValue());
-        assertEquals(BOOL_VARIANT, booleanDetails.getVariant());
-        assertEquals(STATIC_REASON, booleanDetails.getReason());
-
-        stringDetails = api.getClient().getStringDetails(FLAG_KEY_STRING, "wrong");
-        assertEquals(STRING_VALUE, stringDetails.getValue());
-        assertEquals(STRING_VARIANT, stringDetails.getVariant());
-        assertEquals(STATIC_REASON, stringDetails.getReason());
-
-        intDetails = api.getClient().getIntegerDetails(FLAG_KEY_INTEGER, 0);
-        assertEquals(INT_VALUE, intDetails.getValue());
-        assertEquals(INT_VARIANT, intDetails.getVariant());
-        assertEquals(STATIC_REASON, intDetails.getReason());
-
-        floatDetails = api.getClient().getDoubleDetails(FLAG_KEY_DOUBLE, 0.1);
-        assertEquals(DOUBLE_VALUE, floatDetails.getValue());
-        assertEquals(DOUBLE_VARIANT, floatDetails.getVariant());
-        assertEquals(STATIC_REASON, floatDetails.getReason());
-
-        objectDetails = api.getClient().getObjectDetails(FLAG_KEY_OBJECT, new Value());
-        assertEquals(INNER_STRUCT_VALUE, objectDetails.getValue().asStructure()
-                .asMap().get(INNER_STRUCT_KEY).asString());
-        assertEquals(OBJECT_VARIANT, objectDetails.getVariant());
-        assertEquals(STATIC_REASON, objectDetails.getReason());
-    }
+//    @Test
+//    @Disabled
+//    void invalidate_cache() {
+//        ResolveBooleanResponse booleanResponse = ResolveBooleanResponse.newBuilder()
```

Comment:
I'm assuming these aren't meant to be commented?

Reply:
these tests do not compile anymore. discussed with @aepfli. The functionality should be covered by e2e tests. Given the complexity of this test, its not worth to make them work or maintain them. I will delete this. ok?

Reply:
Full support for a delete since they are covered e2e